app/models/Address.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import models.reference.Country
+import play.api.libs.json.{Json, OFormat}
+
+case class Address(
+  streetAndNumber: String,
+  city: String,
+  postalCode: Option[String],
+  country: Country
```

Comment:
We can probably just remove this class and use DynamicAddress now can we?

Reply:
I had it as "Address" with the country held within it because it was cleaner IMO, keeping related data together in a single page but I am happy to make this change